sdk/communication/azure-communication-common/src/main/java/com/azure/communication/common/ConnectionString.java

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.communication.common;
+
+import java.util.Objects;
+
+public class ConnectionString {
```

Comment:
Does not azure core already have an implementation of this that we can use?

Reply:
I checked Azure Core because I saw for C# there is existing implementation, but nothing there for Java unfortunately